pkg/controllers/jivavolume_controller.go

```diff
@@ -54,7 +54,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	openebsiov1alpha1 "github.com/openebs/jiva-operator/pkg/apis/openebs/v1alpha1"
+	api "github.com/openebs/jiva-operator/pkg/apis/openebs/v1"
```

Comment:
n_: for better readability, it may be a good idea to replace "api" with "jivaAPIV1" or "jivaV1"

Reply:
Replaced this with `api` as every time a version change happens we don't have to change the import name in multiple places. This makes the changes simpler for future.